lib/ansible/modules/windows/win_feature.py

```diff
@@ -82,25 +82,27 @@
 '''
 
 EXAMPLES = r'''
-# This installs IIS.
-# The names of features available for install can be run by running the following Powershell Command:
-# PS C:\Users\Administrator> Import-Module ServerManager; Get-WindowsFeature
-$ ansible -i hosts -m win_feature -a "name=Web-Server" all
-$ ansible -i hosts -m win_feature -a "name=Web-Server,Web-Common-Http" all
-ansible -m "win_feature" -a "name=NET-Framework-Core source=C:/Temp/iso/sources/sxs" windows
```

Comment:
This is actually against the "Examples should be written in YAML format in plain text in an EXAMPLES string within the module" present in http://docs.ansible.com/ansible/dev_guide/developing_modules.html#example